internal/work/dispatcher.go

```diff
@@ -2,13 +2,6 @@ package work
 
 import (
 	"fmt"
-	"io"
-	"net/url"
-	"os"
-	"path/filepath"
-	"strings"
-	"time"
-
```

Comment:
This change should not be applied. `goimports` sorts stdlib imports ahead of external packages. They should not be intermixed.